src/sparseml/transformers/sparsification/obcq/obcq.py

```diff
@@ -78,6 +78,9 @@ def one_shot(
     elif "llama" in model_path.lower():
         model_loader_fn = SparseCasualLM.llama_model_from_pretrained
         forward_fn = llama_forward
+    elif "mistral" or "zephyr" in model_path.lower():
```

Comment:
I don't think this evaluates the way you intend since the portion after the `or` is evaluated separately. as a bool would probably always evaluate to `True`

```python
>>> "mistral" or "zephyr" in "mistral"
'mistral'
```

Reply:
this is a good catch! i'm actually going to change this to use the model_type in the config to be more robust